src/lib/registration/ui/migration_repos_workflow.rb

```diff
@@ -43,6 +43,14 @@ def self.run
       def initialize
         textdomain "registration"
 
+        # a dummy message which will be used later, just make sure we have it
+        # before the translation deadline...
+        # TRANSLATORS: popup question, confirm installing the available
+        # updates now
+        _("There are some online updates available to installation,\n" \
```

Comment:
My English may not be best, anyway, here is my suggestion:
There are online update pending for installation.
[...]
Install pending updates now?